### ISSUE TITLE
rule aws_lb_listener_rule redirect path must start with '/'

### ISSUE THREAD

**Issue by metafeather** [terraform-linters/tflint-ruleset-aws#386]
Using: tflint@0.41.0 and ruleset@0.17.1.
Provider docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_listener_rule#redirect

Add an example and error I made that could probably be caught:

```hcl
    redirect {
      host        = "other.example.com"
      port        = 443
      path        = "#{path}"
      query        = "#{query}"
      status_code = "HTTP_301"
    }
```
`Error: Error creating LB Listener Rule: InvalidLoadBalancerAction: The Path parameter must be a valid path, should start with a '/', and may contain up to one of each of these placeholders: '#{path}', '#{host}', '#{port}'.
